week09/movie/src/components/Detail.jsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import styled from "styled-components";
+
+const Title = styled.div`
+    font-size: 40px;
+    margin: 10px;
+`;
+const ImgAndInfo = styled.div`
+    display: flex;
+`;
+const Info = styled.div`
+    display: flex;
+    flex-direction: column;
+    margin: 10px;
+`;
+const Desc = styled.div`
+    font: 10px;
+    margin: 10px;
+`;
+
+const Div = styled.div`
+    font-size: 23px;
+    margin: 10px;
+`;
+
+const Detail = ({ info }) => {
+    const title = info.title;
+    const img = info.medium_cover_image;
+    const year = info.year;
+    const rating = info.rating;
+    const runtime = info.runtime;
+    const genres = info.genres;
+    const description = info.description_full;
+
+    return (
+        <div>
+            <Title>{title}</Title>
+            <ImgAndInfo>
+                <img src={img} style={{ marginLeft: "10px" }} />
+                <Info>
+                    <Div>Year : {year}</Div>
+                    <Div>Rating : {rating}</Div>
+                    <Div>Runtime : {runtime} min</Div>
+                    <Div>Genres : {genres.join(", ")}</Div>
+                </Info>
+            </ImgAndInfo>
+            <Div>Description</Div>
+            <Desc>{description}</Desc>
+        </div>
+    );
```

Comment:
```suggestion
    return (
        <DetailComponent>
          <div>
            <Title>{title}</Title>
            <ImgAndInfo>
              <img src={img} style={{ marginLeft: "10px" }} alt="Movie Poster" />
              <Info>
                <Div>Year : {year}</Div>
                <Div>Rating : {rating}</Div>
                <Div>Runtime : {runtime} min</Div>
                <Div>Genres : {genres.join(", ")}</Div>
              </Info>
            </ImgAndInfo>
            <Div>Description</Div>
            <Desc>{description}</Desc>
          </div>
    </DetailComponent>
    );
```
전체를 DetailComponent로 묶어준 후
```
const DetailComponent = styled.div`
  min-height: 100vh;
`;
```
styled-components 코드를 추가합니다.
이제 정상적으로 화면 전체에 배경색이 로드됩니다.